blueocean-git-pipeline/pom.xml

```diff
@@ -24,8 +24,13 @@
       <artifactId>blueocean-pipeline-scm-api</artifactId>
     </dependency>
     <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>blueocean-pipeline-api-impl</artifactId>
+    </dependency>
+    <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git</artifactId>
+      <version>3.5.1-SNAPSHOT</version>
```

Comment:
@jglick right, I'm just waiting for a new git-plugin release, should be this weekend, I'm told - the people who need to can't easily test this until the git-plugin is published at least to the experimental update center, anyway AFAIK